### PR TITLE
Pull data from identity store

### PR DESCRIPTION
**Pull request by alexmuller**
Apart from the actual usage of the identity store, there are a couple of things to be aware of:

- This is a breaking change because the names for the configuration options have had to change
- There is one minor change in functionality - if there are no results from the hub then the list of items will be displayed "Unknown" instead of not being displayed

### CODE REVIEW

**Review by rudigiesler — APPROVED** [review]
:+1:

I'm not too concerned with the breaking changes, since this is versioned, and we are incrementing the major version here.

File: casepropods/family_connect_registration/plugin.py [hunk 1]
```diff
@@ -20,6 +25,18 @@ class RegistrationPodConfig(PodConfig):
 
 
 class RegistrationPod(Pod):
+    def lookup_field_from_dictionaries(self, field, *lists):
+        """
+        Receives a 'field' and one or more lists of dictionaries
+        to search for the field. Returns the first match.
+        """
+        for results in lists:
```
Comment (KaitCrawford) [inline casepropods/family_connect_registration/plugin.py]: I think these variable names could be a bit more explicit. Like `for list in lists` ` for dict in list` `if field in dict` or something like that. Currently it's a bit difficult to figure out what you're dealing with here.

File: casepropods/family_connect_registration/tests.py [hunk 2]
```diff
@@ -147,3 +226,82 @@ def test_no_http_request_if_contact_uuid_is_none(self):
...
+        result = self.pod.read_data({'case_id': self.case.id})
+
+        self.assertEqual(len(result['items']), 13)
+        self.assertEqual(
+            result['items'][0],
+            {'name': 'Mother Name', 'value': 'sue-toplevel'},
+        )
+
+    @responses.activate
+    def test_hub_data_precendence_over_identity_store(self):
```
Comment (KaitCrawford) [inline casepropods/family_connect_registration/tests.py]: Thinking about it I don't think the hub should have precedence here. A user can change their details outside of a registration so registrations may not be up today. Whereas a new registration should always update the info in the identity store. So the identity store should be considered the source of truth for a user.

**Review by KaitCrawford — COMMENTED** [review]
Looks great! Just two small comments

**Comment by alexmuller** [comment]
@KaitCrawford Thanks, updated to address your comments!

**Review by KaitCrawford — APPROVED** [review]
:+1: :cake:

**Pull request merged**